### ISSUE TITLE
Implement Z and X extensions without rotating

### ISSUE THREAD

**Issue by nelimee** [QCHackers/tqec#53]
The last step before the full "corner" is to be able to perform the following steps:

1. Initialize a logical qubit in the $\vert 0_L\rangle$ state.
2. Perform a few rounds of memory.
3. Move the logical qubit in the $Z$ direction (exactly #50).
4. Perform a few rounds of memory on the newly placed logical qubit.
5. Move the logical qubit in the $X$ direction (exactly #51).
6. Perform a few rounds of memory on the newly placed logical qubit.
7. Measure the observable.

Basically, a corner without the complexity of changing plaquette orientation midway and changing the orientation of the observables.

Idea provided by Austin.
